vxquery-server/src/main/resources/conf/cluster.properties

```diff
@@ -52,3 +52,6 @@ NCJAVA_OPTS="-server -Xmx7G -Djava.util.logging.config.file=./vxquery-benchmark/
 # debug option: NCJAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,address=7002,server=y,suspend=n -Xmx1g -Djava.util.logging.config.file=logging.properties"
 # Yourkit option: -agentpath:/tools/yjp-2014-build-14114/bin/linux-x86-64/libyjpagent.so=port=20001"
 # Yourkit mac option: -agentpath:/Applications/YourKit_Java_Profiler.app/bin/mac/libyjpagent.jnilib=sampling
+
+#HDFS configuration directory
+HDFS_CONF=/home/efi/Utilities/hadoop/etc/hadoop
```

Comment:
This should be a default option. Don't include a path specific to your computer in the repository.

Basically we should have a default and then user can add their settings in a non repo file.